client/ayon_core/modules/deadline/plugins/publish/submit_houdini_render_deadline.py

```diff
@@ -71,6 +71,7 @@ class HuskStandalonePluginInfo():
     PostFrame = attr.ib(default="")
     PostRender = attr.ib(default="")
     RestartDelegate = attr.ib(default="")
+    Version = attr.ib(default="")
```

Comment:
I'm fine with this change here - just wanted to remark that this may have been better as a separate PR since it's a completely isolated change to also submit along with the version.

Reply:
Implemented this upstream currently. Thanks!